src/mumble/MainWindow.cpp

```diff
@@ -741,6 +741,36 @@ ClientUser *MainWindow::getContextMenuUser() {
 	return nullptr;
 }
 
+ContextMenuTarget MainWindow::getContextMenuTargets() {
+	ClientUser *p = nullptr;
```

Comment:
I would simply create a `ContextMenuTarget` object here. Then you don't need two separate variables floating around and the `nullptr` initialization is implicit.